src/main/java/com/dumbdogdiner/stickyapi/bukkit/book/commonmarkextensions/MCFormatExtension.java

```diff
@@ -2,10 +2,13 @@
  * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
  * Licensed under the MIT license, see LICENSE for more information...
  */
-package com.dumbdogdiner.stickyapi.common.book.commonmarkextensions;
+package com.dumbdogdiner.stickyapi.bukkit.book.commonmarkextensions;
 
+import org.commonmark.ext.gfm.strikethrough.internal.StrikethroughDelimiterProcessor;
 import org.commonmark.parser.Parser;
 
+
+
 public class MCFormatExtension implements Parser.ParserExtension {
```

Comment:
Change this to `FormatExtension` 

```suggestion
public class MCFormatExtension implements Parser.ParserExtension {
```